src/pseudopeople/schema_entities.py

```diff
@@ -445,6 +447,11 @@ class __Columns(NamedTuple):
     spouse_household_id: Column = Column(
         "spouse_household_id",
     )
+    spouse_relation_to_reference_person: Column = Column(
```

Comment:
Can you just change this to spouse_relationship_to...? If not I might forget since it won't introduce a merge conflict when you merge.

Reply:
So this will get updated when I have to merge because I will have to rebase off develop but the sample data on this branch is not the sample data you ran. I don't want to rebase right now while the review is up. I will just need to run tests once I have the new sample data merged here..